src/plugins/charts/public/services/palettes/helpers.ts

```diff
@@ -70,7 +70,10 @@ export function workoutColorForValue(
   const comparisonFn = (v: number, threshold: number) => v - threshold;
 
   // if steps are defined consider the specific rangeMax/Min as data boundaries
-  const maxRange = stops.length ? rangeMax : dataRangeArguments[1];
+  // as of max reduce its value by 1/colors.length for correct continuity checks
+  const maxRange = stops.length
+    ? rangeMax
+    : dataRangeArguments[1] - (dataRangeArguments[1] - dataRangeArguments[0]) / colors.length;
```

Comment:
shouldn't this be just `dataRangeArguments[0] / colors.length`? I mean:
`x-(x-y) = x-x+y = y`.

Reply:
The division has precedence over subtraction. 

It basically does `x - (x-y)/z` (i.e. `100 - (100 - 0)/5` => `100 - 20 = 80`)

Reply:
Oh, I thought it is in the brackets, nvm then!